back-end/student-tracker-template.yaml

```diff
@@ -198,6 +208,34 @@ Resources:
           - LambdaFuncNameArn: !GetAtt LambdaFunction.Arn
     DependsOn:
       - APIGatewayResource
+  APIGatewayCoursesGetMethod:
+    Type: AWS::ApiGateway::Method
+    Properties:
+      RestApiId: !Ref APIGatewayRestAPI
+      ResourceId: !Ref APIGatewayResource
```

Comment:
This needs to be the courses reference you set up...
  APIGatewayCoursesResource

Reply:
Ok. so i should change the !Ref APIGatewayResource to APIGatewayCoursesResource